<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Kreis</class>
 <widget class="QDialog" name="Dialog_Kreis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kreis</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_x">
      <property name="text">
       <string>Mittelpunkt X-Wert</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_y">
      <property name="text">
       <string>Mittelpunkt Y_Wert</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_z">
      <property name="text">
       <string>Mittelpunkt Z-Wert</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_r">
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_x"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_y"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_z"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_r"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>311</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_x</tabstop>
  <tabstop>lineEdit_y</tabstop>
  <tabstop>lineEdit_z</tabstop>
  <tabstop>lineEdit_r</tabstop>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_Abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
